composer.json

```diff
@@ -29,14 +29,14 @@
     ],
     "scripts": {
         "test": "phpunit",
-        "test:coverage": "vendor/bin/phpunit --coverage-html coverage"
+        "test:coverage": ["@putenv XDEBUG_MODE=coverage","vendor/bin/phpunit --coverage-html coverage"]
     },
     "require": {
-        "php": ">=7.1.0"
+        "php": ">=7.4.0"
```

Comment:
Dit is een breaking change. In de README staat dat we momenteel PHP 7.1 ondersteunen. Voor klanten die < 7.4 hebben zou dan ineens de package niet meer installable of updateable zijn.

Reply:
Ja weet even niet waarom deze is aangepast, zal ik terugzetten.